tests/hpc/slurm_master.pm

```diff
@@ -483,4 +505,15 @@ sub post_fail_hook ($self) {
     upload_logs('/var/log/slurmctld.log');
 }
 
+sub create_user_account ($username) {
+    return 1 unless $username;
```

Comment:
I'd expect you need to do `my $username = shift;`

Reply:
in fact i dont even need this check due to `-signatures` the function will complain when the signature is not invoked with `$username`. i will remove it

Reply:
Done